bundles/org.openhab.binding.omnikinverter/src/main/java/org/openhab/binding/omnikinverter/internal/OmnikInverterMessage.java

```diff
@@ -194,4 +194,12 @@ public double getEnergyToday() {
     public double getTotalEnergy() {
         return getInt(71, 10);
     }
+
+    /**
+     *
+     * @return the current temprature in Celcius
+     */
+    public double getTemperature() {
+        return getShort(31, 10);
+    }
```

Comment:
Can you please add test coverage for this method in [OmnikInverterMessageTest](https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.omnikinverter/src/test/java/org/openhab/binding/omnikinverter/internal/test/OmnikInverterMessageTest.java)?

Reply:
I added test support for the temperature in the commit "[omnikinverter] Added test coverage for temperature"